src/main/java/com/vistatec/ocelot/Ocelot.java

```diff
@@ -514,4 +563,40 @@ public void actionPerformed(ActionEvent e) {
             });
         }
     }
+    
+    public static class MyOwnFocusTraversalPolicy extends FocusTraversalPolicy {
```

Comment:
Is this class used?  I don't see any calls to `setFocusTraversalPolicy()`.  If not, we should clean it up.